### PR TITLE
Guide to Open Source Licenses

### PR DESCRIPTION
**Pull request by lalith1403**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [ ] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by lalith1403** [comment]
This fixes #708

File: articles/open-source-licencses/index.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+5. Creative Commons Attribution
+6. The Unlicense
+7. Codebases with no license included
+
+### How to license
+
+After going through this article, you will know which license would be the best fit for your use case. Whichever license you choose, you must include the license along with the source code. There are a couple of ways to ensure the license is included.
+
+1. Go to the link of the licencse page and find the license. Include a link to the license and mention that the work presented is licensed under the respective license.
+2. The second way is to include the markdown version of the license in a file called LICENSE.md. GitHub recognizes this file and displays the type of license on the right panel of the repository page.
```
Comment (cyberShaw) [inline articles/open-source-licencses/index.md:22]: >  LICENSE.md

This is being recognized as a link by markdown and is redirecting to a website named license.md when I click on it. Could you please make sure this doesn't happen, perhaps by adding backticks?

```md
`license.md`
```

File: articles/open-source-licencses/index.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+
+#### Limitations
+- No warranty or liability is provided
+
+
+### Codebase with no license included
+
+Many times the author or contributors may release the code without any license file attached with it. In such cases, one should contact the contributors and ask them to include a license. This would ensure minimal legal conflicts, if any, in the future. Until a license is included, the author or contributors have exclusive rights over the code. One may not copy or use it without their permission. 
+
```
Comment (cyberShaw) [inline articles/open-source-licencses/index.md:104]: Great article, @lalith1403! One suggestion that would make this article better would be the addition of a summary table, where you compare the permissions, limitations and "ideal for" column, that discusses the licenses in a crisp manner.

**Review by cyberShaw — CHANGES_REQUESTED** [review]
Good one as always, @lalith1403! Please make the suggested changes so that we can move on to final review. 💯

**Comment by lalith1403** [comment]
@cyberShaw , I have made the requested changes. Thank you.

**Comment by cyberShaw** [comment]
Thank you for making the changes, @lalith1403!

**Comment by hectorkambow** [comment]
plagiarism check done

**Pull request merged**